java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -310,6 +310,37 @@ public BaseDeviceMemoryBuffer getDeviceBufferFor(BufferType type) {
     return srcBuffer;
   }
 
+  /**
+   * Create a ColumnVector from the off heap Apache Arrow buffers passed in.
```

Comment:
Nit: Docs should explain how to pass missing buffers to this API.

Reply:
will add